src/thriveopedia/pages/ThriveopediaPatchMapPage.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using Godot;
+
+/// <summary>
+///   Thriveopedia page displaying the patch map for the current game.
+/// </summary>
+/// <remarks>
+///   <para>
+///     Note a lot of this functionality is duplicated from PatchMapEditorComponent.
```

Comment:
For cleaner code would it be possible to create a new class that is used by composition by both this and PatchMapEditorComponent so that most of the tricky logic could be included there (I also saw that this at least duplicates the seed display scene part, so maybe that new class should be a higher level wrapper scene on the underlying map component to provide the common functionality)?

Reply:
I've just spent several hours trying to get this to work and gave up. There's so much interdependence in the editor classes that I couldn't separate neatly.

I got it working but saves were broken, and any attempt to fix it created horrid spaghetti code. Honestly making such a major change to the editor would introduce so much more risk for bugs and extra review requirements that I doubt it's a good idea for me to do it here.

If you want to fix it go ahead, but I don't think I can.

Reply:
Sounds like a pain. I feel like someone has finally experienced the pain I had with trying to setup the later stage scenes when I needed to refactor almost everything in the game and HUD scenes...

We can open a follow up issue for this, though until that is done people need to be careful not to accidentally make changes in only one place the map is shown.